app/views/site/study.html.erb

```diff
@@ -17,7 +17,7 @@
         <%= render partial: 'study_download_data' %>
       </div>
     <% end %>
-    <% if @allow_firecloud_access && @allow_computes && @user_can_compute %>
+    <% if @allow_firecloud_access && @user_can_compute %>
```

Comment:
Nice simplification!

Reply:
Unfortunately I had to take it out because I realized that it was there specifically for thenav-tabs.  But I did make it so that it can source existing calls rather than make a new one.